client/state/rewind/selectors/get-backup-browser-selected-list.ts

```diff
@@ -0,0 +1,87 @@
+import { BackupBrowserItem, BackupBrowserSelectedItem } from 'calypso/state/rewind/browser/types';
+import { AppState } from 'calypso/types';
+
+const getNodeFullPath = ( node: BackupBrowserItem ): string => {
+	let fullPath = node.ancestors.join( '/' ) + '/' + node.path;
+	if ( node.ancestors[ 0 ] === '/' ) {
+		fullPath = fullPath.slice( 1 );
+	}
+	return fullPath;
+};
+
+// Recursive function to iterate through tree and add items to the list
+const addChildrenToList = (
```

Comment:
This looks very similar to what we have in `client/state/rewind/selectors/get-backup-browser-check-list.ts`, I think might be worth to reuse the same logic to prepare the included items.

Reply:
Yeah, the main difference is the exclusion logic.  We could pull it out into a common util and pass some `noExclusion`.  On that point, do you have an idea for where to put common selector utilities?  We could potentially put them in the same file as well, but there seems to be a convention for single selectors per file.

Reply:
> On that point, do you have an idea for where to put common selector utilities? We could potentially put them in the same file as well, but there seems to be a convention for single selectors per file.

We can create utils/helpers in `client/state/rewind/browser` dir and then import them into your selectors.

Reply:
These two ended up different enough that I'm just leaving it as is.  Beyond the naming they're populating different types of objects, taking different params and outputting different types.